src/Common/Normalizer/MigratingNormalizer.php

```diff
@@ -23,13 +23,13 @@ public function __construct(
 
     public function normalize(mixed $value, string $typeName): mixed
     {
-        $typeMigrations = $this->typeToMigrationsMap[$typeName] ?? null;
+        $migrations = $this->typeToMigrationsMap[$typeName] ?? null;
```

Comment:
Unrelated change. Just spotted this naming inconsistency with method denormalize.